composer.json

```diff
@@ -37,6 +37,7 @@
     "extra": {
         "_ezplatform_branch_for_behat_tests": "master",
         "branch-alias": {
+            "dev-custom_tag_content_edit": "1.4.x-dev",
```

Comment:
You should remove this line from this branch as composer with current meta setup is treating it as a dev-master (latest version).
ezsystems/ezplatform-admin-ui                      1.4.x-dev c69ca0d  eZ Platform Admin v2

It is messes up development process during certification.

Reply:
Done. Sorry about that :)

Reply:
Thank you :)